### PR TITLE
Script & doc tweaks to ease running frontend tests in Docker

### PR DESCRIPTION
**Pull request by lmorchard**
I'm still doing my gulp frontend builds in Docker pending a resolution of #359, so this PR makes it easy to run the frontend budo test runner in that frontend watcher Docker container.

### CODE REVIEW

File: README-DOCKER.md [hunk 1]
```diff
@@ -19,16 +19,22 @@ lists some hints & tips we've accumulated through the course of daily work.
...
+    * To run frontend & addon code linting:
 
-      `docker exec -t -i testpilot_frontend_watcher_1 ./bin/run-frontend-tests.sh`
+      `docker exec testpilot_frontend_watcher_1 ./bin/run-frontend-lint.sh`
+
+    * To start a file watcher for frontend tests:
+
+      `docker exec testpilot_frontend_watcher_1 ./bin/run-frontend-tests-watch.sh`
+
+      Then, open http://ideatown.dev:9966/ in a browser to see test results.
```
Comment (pdehaan) [inline README-DOCKER.md]: change from "http://ideatown.dev/" to "http://testpilot.dev/"?

File: .gitignore [hunk 2]
```diff
@@ -24,3 +24,5 @@ docker-compose-s3.yml
 !.elasticbeanstalk/*.global.yml
 .ebextensions/00-environment.config
 Dockerrun.aws.json
+
+debug-config.json
```
Comment (meandavejustice) [inline .gitignore:28]: nit: add this change to the commit message

**Comment by meandavejustice** [comment]
down to merge this after these two comments are addressed

**Comment by lmorchard** [comment]
Updated the commit

**Pull request merged**